test/system/issues_test.rb

```diff
@@ -113,7 +113,7 @@ def test_issue_index_with_multiple_roles
 
     visit issues_path
 
-    assert page.has_link?(I18n.t("issues.index.reports_count", :count => issue1.reports_count), :href => issue_path(issue1))
-    assert page.has_link?(I18n.t("issues.index.reports_count", :count => issue2.reports_count), :href => issue_path(issue2))
+    assert_link I18n.t("issues.index.reports_count", :count => issue1.reports_count), :href => issue_path(issue2)
```

Comment:
Why does this change from `issue1` to `issue2`?

Reply:
Great catch! That was a mistake caused by me messing around trying to get the friendly error message to show up. I've force-pushed the corrected version.